test/eca/features/chat_tool_call_state_test.clj

```diff
@@ -0,0 +1,944 @@
+(ns eca.features.chat-tool-call-state-test
```

Comment:
I usually prefer test ns to follow the same ns name, and have deftests per function being tested with multiple testings, unless too complex (which looks like this one). Wondering if we should not move those tests to eca.features.chat-test

Reply:
Ok, let's keep it here and It becomes weird we move to chat later.

Reply:
OK, thanks.

Would you please take a look at master?
The integration tests are failing again on github and also in my repo after updating the latest github master.
The integration tests are failing in the same way as the last time.
I have only added tests, not changed the source code.
So, I'm guessing the problem is the same as last time?
Please check and let me know.

Thanks for your help.

Reply:
@bombaywalla integration tests at master are now passing (tested local as well), so I believe it's related with your PR or some merge with master?

Reply:
Thanks for checking.

I can't quite figure out what is wrong, since I only added unit tests and did not change code.  I did `git push origin --force-with-lease` to push the branch.  That's the only thing I can currently think of that causes this problem.  Anyways, I will investigate more.

Reply:
Maybe try merging master again? I can take a look tomorrow if you don't find it.

Reply:
It worked after merging and rebasing off the latest changes to master today.